### ISSUE TITLE
FallbackIconGenerator fails if nextcloud is installed in a subdirectory

### ISSUE THREAD

**Issue by eglia** [marius-wieschollek/passwords#135]
### System Information
- Nextcloud Version: 15.0.7
- PHP Version: 7.2.15
- Passwords Version: 2019.4.0
- Server OS and Version: Ubuntu 18.04
- Webserver: nginx 1.14.0

### Steps to reproduce
1. Install nextcloud to a subdirectory (e.g. example.com/nextcloud)
2. Open the passwords app
3. create an entry for which the fallback icon is generated

### Actual result
The fallback icon fails to generate because it can't find the default font. It is looking
for the following file:
/var/www/html/core/fonts/Nunito-Regular.ttf

### Expected result
The font is actually located under:
 /var/www/html/nextcloud/core/fonts/Nunito-Regular.ttf

### Nextcloud log
<details>
<summary>Nextcloud log</summary>

```
{"reqId":"RmGAdhsmCSfERtAMn0Db","level":3,"time":"2019-04-11T10:21:19+02:00","remoteAddr":"XXX","user":"YYY","app":"passwords","method":"GET","url":"/nextcloud/index.php/apps/passwords/api/1.0/service/favicon/opensubtitles.org/32","message":{"Exception":"ImagickException","Message":"The path does not exist: /var/www/html/core/fonts/Nunito-Regular.ttf","Code":1,"Trace":[{"file":"/var/www/html/nextcloud/apps/passwords/lib/Helper/Image/ImagickHelper.php","line":108,"function":"setfont","class":"Imagick","type":"->","args":["/var/www/html/core/fonts/Nunito-Regular.ttf"]},{"file":"/var/www/html/nextcloud/apps/passwords/lib/Helper/Icon/FallbackIconGenerator.php","line":124,"function":"getImageFromFile","class":"OCA\\Passwords\\Helper\\Image\\ImagickHelper","type":"->","args":["/tmp/5caef8ff877fd.svg"]},{"file":"/var/www/html/nextcloud/apps/passwords/lib/Helper/Icon/FallbackIconGenerator.php","line":81,"function":"createIconFromSvg","class":"OCA\\Passwords\\Helper\\Icon\\FallbackIconGenerator","type":"->","args":["#9b59b6","O",32]},{"file":"/var/www/html/nextcloud/apps/passwords/lib/Helper/Favicon/AbstractFaviconHelper.php","line":97,"function":"createIcon","class":"OCA\\Passwords\\Helper\\Icon\\FallbackIconGenerator","type":"->","args":["opensubtitles.org",32]},{"file":"/var/www/html/nextcloud/apps/passwords/lib/Services/FaviconService.php","line":96,"function":"getDefaultFavicon","class":"OCA\\Passwords\\Helper\\Favicon\\AbstractFaviconHelper","type":"->","args":["opensubtitles.org",32]},{"file":"/var/www/html/nextcloud/apps/passwords/lib/Controller/Api/ServiceApiController.php","line":180,"function":"getFavicon","class":"OCA\\Passwords\\Services\\FaviconService","type":"->","args":["opensubtitles.org",32]},{"file":"/var/www/html/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":166,"function":"getFavicon","class":"OCA\\Passwords\\Controller\\Api\\ServiceApiController","type":"->","args":["opensubtitles.org",32]},{"file":"/var/www/html/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":99,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Passwords\\Controller\\Api\\ServiceApiController"},"getFavicon"]},{"file":"/var/www/html/nextcloud/lib/private/AppFramework/App.php","line":118,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Passwords\\Controller\\Api\\ServiceApiController"},"getFavicon"]},{"file":"/var/www/html/nextcloud/lib/private/AppFramework/Routing/RouteActionHandler.php","line":47,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["ServiceApiController","getFavicon",{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},{"domain":"opensubtitles.org","size":"32","_route":"passwords.service_api.get_favicon"}]},{"function":"__invoke","class":"OC\\AppFramework\\Routing\\RouteActionHandler","type":"->","args":[{"domain":"opensubtitles.org","size":"32","_route":"passwords.service_api.get_favicon"}]},{"file":"/var/www/html/nextcloud/lib/private/Route/Router.php","line":297,"function":"call_user_func","args":[{"__class__":"OC\\AppFramework\\Routing\\RouteActionHandler"},{"domain":"opensubtitles.org","size":"32","_route":"passwords.service_api.get_favicon"}]},{"file":"/var/www/html/nextcloud/lib/base.php","line":987,"function":"match","class":"OC\\Route\\Router","type":"->","args":["/apps/passwords/api/1.0/service/favicon/opensubtitles.org/32"]},{"file":"/var/www/html/nextcloud/index.php","line":42,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"/var/www/html/nextcloud/apps/passwords/lib/Helper/Image/ImagickHelper.php","Line":108,"CustomMessage":"--"},"userAgent":"Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36","version":"15.0.7.0","id":"5caef8ffcca34"}
```
</details>

### Nextcloud configuration
 'overwritewebroot' => '/nextcloud'

**Comment by marius-wieschollek** [comment]
I have released a nightly update with a fix for this. It would be great if you could check it.

You can install nightly updates by enabling the option Show Nightly Updates in "Apps" in the Passwords admin settings.

**Comment by eglia** [comment]
That was quick!
I can confirm that the issue is fixed in 2019.4.1-build2809
